ssrc_config/config_hitl_eth_gzsim_fw.txt

```diff
@@ -0,0 +1,118 @@
+# Default parameter set for HITL with ethernet Gazebo Sim connection on Skywalker X8 FW
+# [ type: hitl_eth_gzsim ]
+
+param set SYS_AUTOSTART 4421
+param set SYS_AUTOCONFIG 0
+
+####################################
+# general HITL configuration
+####################################
+
+# Set HITL related flag
+param set SYS_HITL 1
+param set MAV_HITL_SHOME 1
+param set SENS_EN_GPSSIM 1
+param set SENS_EN_BAROSIM 1
+param set SENS_EN_MAGSIM 1
+param set SENS_EN_ARSPDSIM 1
+
+# Disable safety switch
+param set COM_PREARM_MODE 0
+param set CBRK_IO_SAFETY 22027
+
+# Disable RC loss failsafe check
+param set NAV_RCL_ACT 0
+
+# Disable RC input requirement
+param set COM_RC_IN_MODE 1
+
+# EKF2
+param set EKF2_MULTI_IMU 3
+
+# Sensor
+param set CAL_ACC0_ID  1310988
+param set CAL_GYRO0_ID 1310988
+param set CAL_ACC1_ID  1310996
+param set CAL_GYRO1_ID 1310996
+param set CAL_ACC2_ID  1311004
+param set CAL_GYRO2_ID 1311004
+param set CAL_MAG1_ID 197388
+param set CAL_MAG1_PRIO 50
+
+
+####################################
+# specific fw HITL configuration
+####################################
+
+# Override airframe defaults
+param set ASPD_DO_CHECKS 7
+param set FW_LAUN_DETCN_ON 0
+
+# HITL PWM functions
+param set HIL_ACT_FUNC1 201
+param set HIL_ACT_DIS1 500
+param set HIL_ACT_FUNC2 202
+param set HIL_ACT_DIS2 500
+param set HIL_ACT_FUNC4 101
```

Comment:
Could you pleas explain, why we use here reverse order (the servos are  the first and the motor is the last)?
In airframe there is right order https://github.com/tiiuae/px4-firmware/blob/main/ROMFS/px4fmu_common/init.d-posix/airframes/4440_gz_ssrc_skywalker_x8#L40

Reply:
Actually the order there doesnt matter. I did that so it's similar to the order defined in the airframe https://github.com/tiiuae/px4-firmware/blob/6e6f3b146e31c34b20ab0e41321c35671eb7a817/ROMFS/px4fmu_common/init.d/airframes/4421_ssrc_skywalker#L23

Reply:
I ment why the pusher motor for sitl has one **FUNC** number ` SIM_GZ_EC_FUNC1 101` and for the real drone has 
 another **FUNC** number `PWM_MAIN_FUNC4 101`.

Reply:
I think there is no intention behind using FUNC1 in SITL and FUNC4 in real / HITL drone. For real / HITL drone, that order (using FUNC4) is from upstream. For SITL drone, it's just natural to assign the 1st FUNC from SIM_GZ_EC to the only motor in FW.

Reply:
Thank you very much for your explanation! It was valuable for me!